idunn/utils/categories.yml

```diff
@@ -31,7 +45,7 @@ categories:
             - "pharmacy,*"
         regex: "pharmac"
     supermarket:
-        pj_what: "supermarchés hypermarchés"
+        pj_what: "supermarchés"
```

Comment:
"alimentation" could possibly used instead, to include more POIs such as groceries.